e2e-tests/playwright/tests/functional/channels/settings/notifications/highlight_without_notification.spec.ts

```diff
@@ -40,19 +40,19 @@ test('MM-T5465-1 Should add the keyword when enter, comma or tab is pressed on t
     const keywordsInput = await channelPage.settingsModal.notificationsSettings.getKeywordsInput();
 
     // # Enter keyword 1
-    await keywordsInput.type(keywords[0]);
+    await keywordsInput.fill(keywords[0]);
```

Comment:
why is one recommended over the other?

Reply:
Good question and I missed to mention. `type` is for deprecation and Playwright suggest to use `fill` - https://playwright.dev/docs/api/class-keyboard#keyboard-type